palantirnet/federated-search-demo
# Fs 66 update core

## JIRA Ticket(s)

- See ticket https://palantir.atlassian.net/browse/FS-66

## Description

Update core and dependencies to 8.8.5.

## Testing instructions

1. Run 'composer install' from the d8 directory
1. Ensure the latest versions of modules are installed.

## Additional Notes:

For some reason, this keeps installing 8.8.2?


---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
